Makefile

```diff
@@ -102,11 +102,28 @@ bump:
 ship:
 		pod spec lint --allow-warnings DatadogSDK.podspec
 		pod trunk push --allow-warnings --synchronous DatadogSDK.podspec
-		pod repo update
 		./tools/standalone-binary-distro/make_distro_builds.sh
-		@echo "⚠️ DatadogSDKObjc.podspec needs to be tried after ~1 hour:"
-		@echo "pod spec lint --allow-warnings DatadogSDKObjc.podspec"
-		@echo "pod trunk push --allow-warnings DatadogSDKObjc.podspec"
+ifeq (${ci}, true)
+		curl https://app.bitrise.io/app/$$BITRISE_APP_SLUG/build/start.json \
```

Comment:
These API is [now deprecated](https://discuss.bitrise.io/t/deprecation-of-build-trigger-api-on-www-subdomain-use-app-subdomain-or-the-public-api-instead/4721/2) and we should use [`POST /apps/{app-slug}/builds` instead](https://api-docs.bitrise.io/#/builds/build-trigger).